server/src/test/java/org/opensearch/index/mapper/KeywordFieldTypeTests.java

```diff
@@ -256,6 +257,26 @@ public void testMockTermsSortedQuery() {
         }
     }
 
+    @AwaitsFix(bugUrl = "nocommit")
+    public void testHeavyWeight() {
```

Comment:
One more nocommit test. Just a kind of smoke test benchmark. 

In my laptop it runs 14 sec vs 8 sec (shuffled vs sorded).

Reply:
I feel this should not be part of unit test by default, as it takes sometime to run?

Reply:
Right. I just use it as a demo for review. And then will remove before merging.